### PR TITLE
python: Fixes for dev-lang/python-freethreading support

### PR DESCRIPTION
**Pull request by mgorny**
gentoo/gentoo#38918

### CODE REVIEW

File: src/pkgcheck/checks/python.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def is_python_interpreter(pkg):
         # ignore python:2.7 deps since they are being phased out from eclass
         # support
         return pkg.slot is None or not pkg.slot.startswith("2")
-    return pkg.key in ("dev-python/pypy3",)
+    return pkg.key in ("dev-python/pypy3", "dev-lang/python-freethreading")
```
Comment (thesamesam) [inline src/pkgcheck/checks/python.py:55]: Don't we need to update this to pypy3_10 and so on?

Comment (mgorny) [reply src/pkgcheck/checks/python.py:55]: No, we depend on `dev-python/pypy3:=`.

Comment (thesamesam) [reply src/pkgcheck/checks/python.py:55]: Thanks. A comment may help but not required.

**Comment by mgorny** [comment]
Let's put this on hold per https://archives.gentoo.org/gentoo-dev/d3fb6b685e511bce4183b674dd171c9e710765d8.camel@gentoo.org/T/#u. Perhaps we'll go straight for `dev-lang/python3_13t` instead.

**Review by arthurzam — APPROVED** [review]
While I understand this is a draft, I see no issue with the way it is done here. I'm also fine with not touching tests for that :)

**Comment by mgorny** [comment]
Given all the opposition on the mailing list, I retract this. Let's shove freethreading into a slot of `dev-lang/python`. Fewer problems for me, can merge it right away and slot lovers get what they want.

**Pull request closed without merging**